value.go

```diff
@@ -43,7 +43,7 @@ var (
 
 // ToValue will convert an interface{} value to a value digestible by otto/JavaScript
 //
-// This function will not work for advanced types (struct, map, slice/array, etc.) and
+// This function will not work for advanced types (struct, map, slice/array, func, etc.) and
```

Comment:
Ok I see why you mean, can you clarifying in the commit / PR so others don't miss-understand this too :)